etl.js

```diff
@@ -20,6 +20,7 @@ let store = JsonStorage.create(localStorage, "trello-gh-projects", {
 
 let Transform = require("./lib/transform.js");
 let gh = require("./lib/gh.js");
+let trelloFields = require("./trello-fields.json");
```

Comment:
Will you document this in the README similarly to how members.json is documented?